src/pages/homepage/Homepage.jsx

```diff
@@ -34,26 +59,40 @@ export default function Homepage() {
 
   const handleFilter = (event) => {
     event.preventDefault();
-    navigate(`/articles/${event.target.value}`);
+    navigate(`/articles/${event.target.value}${location.search}`);
   };
 
   const handleLimit = (event) => {
     event.preventDefault();
     setLimitNumber(event.target.value);
   };
 
+  const handleSortBy = (event) => {
+    event.preventDefault();
+    const queryObject = objectify(location.search);
+    queryObject.sort_by = event.target.value;
+    const queryString = stringify(queryObject);
+    const query = location.pathname + queryString;
+    navigate(query);
+  };
+
+  const handleOrderBy = (event) => {
+    event.preventDefault();
+    const queryObject = objectify(location.search);
+    queryObject.order_by = event.target.value;
+    const queryString = stringify(queryObject);
+    const query = location.pathname + queryString;
+    navigate(query);
+  };
+
   return (
     <div className={styles.container}>
       <h2>Northcoders Articles</h2>
 
       <div className={styles.header}>
         <div className={styles.dropDownContainer}>
           <h4>Number of articles:</h4>
-          <select
-            defaultValue="5"
-            onChange={handleLimit}
-            className={styles.dropdown}
-          >
+          <select defaultValue="5" onChange={handleLimit} className={styles.dropdown}>
```

Comment:
These sorting options are very hard to use on a mobile screen, you should add a bit more responsive styling so your website can be viewed on mobile devices 📱